cdap-cli/src/main/java/co/cask/cdap/cli/command/adapter/ListAdaptersCommand.java

```diff
@@ -51,12 +51,13 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
     List<AdapterDetail> list = adapterClient.list();
 
     Table table = Table.builder()
-      .setHeader("name", "description", "template", "config")
+      .setHeader("name", "description", "template", "config", "schedule")
```

Comment:
How do I determine which properties should be listed from here though? How about having a `properties` column which has stuff like "schedule=.." if schedule is not null and "instances=.." if instances is not null?